cmd/prometheus/main.go

```diff
@@ -86,6 +86,10 @@ var (
 		Name: "prometheus_config_last_reload_success_timestamp_seconds",
 		Help: "Timestamp of the last successful configuration reload.",
 	})
+	readyStatus = prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
This might be a good candidate to use `promauto` so we don't wait to register it.

Reply:
It does not change anything since we start the HTTP handler after registration anyway.

Reply:
It does help a bit of "forgot to register". :)